modules/web-mercator/src/web-mercator-viewport.d.ts

```diff
@@ -156,5 +158,8 @@ export default class WebMercatorViewport {
    *    [x, y] measured in pixels.
    * @returns {WebMercatorViewport}
    */
-  fitBounds(bounds, options?: {padding?: number; offset?: [number, number]}): WebMercatorViewport;
+  fitBounds(
+    bounds: [[number, number], [number, number]],
+    options?: Omit<FitBoundsOptions, 'width' | 'height' | 'bounds'>,
```

Comment:
The implementation calls the standalone `fitBounds` so this type seems accurate and matches the tests. It doesn't agree w/ the jsdoc, though.